tests/unit_tests/test_predictor.py

```diff
@@ -82,3 +82,27 @@ def test_learn_and_predict_sklearn_mixer(self):
 
         df = df.drop([x['name'] for x in config['output_features']], axis=1)
         predictor.predict(when_data=df)
+
+    def test_text_encoder(self):
+        """
+        2021.02.15
+        Check if the text encoder is properly deployed; NOT performance.
+
+        Config ensures "input" is treated as text.
+        """
+        config = {
+            "input_features": [
+                {"name": "input", "type": "text"},
+            ],
+            "output_features": [
+                {"name": "output", "type": "numeric"},
+            ],
+            "mixer": {"class": BoostMixer},
```

Comment:
Ideally use `LightGBM` or `NnMixer` here, as `BoostMixer` might be deprecated in the near future